arangod/Cluster/ClusterMethods.cpp

```diff
@@ -2712,10 +2706,9 @@ Result flushWalOnAllDBServers(ClusterFeature& feature, bool waitForSync,
   }
 
   for (Future<network::Response>& f : futures) {
-    network::Response const& r = f.get();
-
-    if (r.fail() || r.statusCode() != fuerte::StatusOK) {
-      return r.combinedResult();
```

Comment:
It's worth nothing:
- `r.fail()` would definitely get included in the `combinedResult()`
- `r.statusCode()` here is compared to `StatusOK`. However, `combinedResult()` considers every response code in the interval [200, 300) as successful.

This is just a remark, I don't think this change is going to cause any problems.